examples/systemd/post-install

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# this post upgrade script is run each time the teleport package is upgraded
+
+set -eu
+
+case "$1" in
+    configure)
```

Comment:
We can use `teleport-update link` to accomplish this instead. It will create links from `/usr/local/teleport-system/bin` to `/usr/local/bin` without overriding auto-update-managed links.

Reply:
Yes gonna replace to this one, was testing in isolated docker container initially, actually I opened draft then recall that it hast to be teleport-update instead